src/Hedgehog.Experimental/Gen.fs

```diff
@@ -342,6 +343,7 @@ module GenX =
     }
 
   let defaults =
+    let dateTimeRange = Range.constantFrom (DateTime (2000, 1, 1)) DateTime.MinValue DateTime.MaxValue
```

Comment:
Same as mentioned in the first comment; I'd like an exponential range here.

Reply:
Yep!